distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itest/platform/TestSecurity.java

```diff
@@ -562,7 +568,7 @@ void restoreKeystoreFile() throws IOException {
     //at runtime. The actual functionality of these operations is proved in unit tests.
     @Test
     public void testCertificateGeneratorService() throws Exception {
-        String commonName = "pangalactic";
+        String commonName = "myCn";
```

Comment:
it was triggering our githooks check. since there was no need for it to be anything specific i just changed it to something that wouldn't trigger the check.